### DISCUSSION TITLE
Estimated Allowed Amount determination

### DISCUSSION THREAD

**Discussion by brmhbrul** [CMSgov/hospital-price-transparency#137] [Q&A] [answered]
Does the estimated allowed amount account for deductibles, co-payments, coinsurance, etc? It seems the term "allowed amount" would refer to the maximum amount an insurance will pay for an item or service regardless of the deductible/etc, but the CMS definition for estimated allowed amount specifically states "average dollar amount that the hospital has historically **received**" indicating this is actual payments, not just allowed amounts? If actual payments, the values could vary widely even for the same payer depending on the individual patients' deductibles?  If the insurance allowed $10,000, paid $8,000 and assigned the remaining $2,000 to patient responsibility, what would I use to calculate "estimated allowed amount"? 

I note as well that the article https://github.com/CMSgov/hospital-price-transparency/discussions/113 states "including any adjustments made to the claim, such as denials, reductions, or increases in payment" making me wonder if the intention is actual payments vs. allowed amount. (But I can see how denials would be different than adjustments for patient responsibility.)

Thank you for clarifying, since it seems I'm receiving conflicting information.

**Comment by RidenourK** [comment, marked as answer]
@brmhbrul Thank you for your question. 

In the 2024 OPPS/ASC Final Rule, we indicated that a hospital would only be required to calculate an estimated allowed amount, in dollars, when the hospital has established a payer-specific negotiated charge that can only be expressed as a percentage or an algorithm. This algorithm or percentage is based on the contract the hospital has with a particular payer for a particular plan, and the estimated allowed amount would be the average reimbursement in dollars that it has received from the payer in the past, that is, what some might call an `historical allowed amount.' This estimated allowed amount is therefore not prospective and is also not based on the hospital's chargemaster or claims submitted to the payer which, as we understand it, contains only gross charges for itemized items and services. Because the “expected allowed amount” data element is meant to provide an estimate of what the algorithm produces in dollars, across the group of people covered by a particular payer's plan, we clarify that such an amount should reflect the amount the hospital expects to be reimbursed for the item or service (or service package), on average. To avoid confusion, we modified the definition to refer to the average amount `historically received' (rather than `expects to be paid', and also renamed the data element “estimated allowed amount.”  Please refer to the discussion at [88 FR 82101](https://www.federalregister.gov/d/2023-24293/p-4991).
